src/components/CardPassword/CardPasswordInput.stories.tsx

```
@@ -0,0 +1,33 @@
+
+export default meta;
+type Story = StoryObj<typeof CardPasswordInput>;
+
+export const Default: Story = {
+  args: {
+    cardPassword: '',
+    hasError: false,
+  },
+};
+
+export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)